### DISCUSSION TITLE
Typebot

### DISCUSSION THREAD

**Discussion by gyancarlodasilva** [coollabsio/coolify#2306] [Service Template Requests]
Add a typebot template in coolify 
https://docs.typebot.io/self-hosting/deploy/docker

**Comment by danielehrhardt** [comment]
Someone already added this?
https://docs.typebot.io/self-hosting/get-started

**Comment by louanfontenele** [comment]
+1

It would be cool to add the Typebot system to Coolify, besides being an excellent chatbot creation software, it is excellent for creating automatic messages for Whatsapp even without needing the Whatsapp API, it uses WhatsApp web to trigger the messages.

Docs: https://docs.typebot.io/self-hosting/

Webiste: https://typebot.io/

**Comment by louanfontenele** [comment]
Typebot with EvolutionAPI for Whatsapp

**Comment by tff2011** [comment]
Please add this: Typebot with EvolutionAPI for Whatsapp

**Comment by OrtoQBank** [comment]
update on this?
